### PR TITLE
VOXL2 companion computer

### PR DESCRIPTION
**Pull request by katzfey**
Added VOXL2 to the list of companion computers

### CODE REVIEW

**Comment by rmackay9** [comment]
Hi, this looks very similar to PR https://github.com/ArduPilot/ardupilot_wiki/pull/5846 so I guess I'll post the same comment

I think we should instead either update our [existing page for the VOXL1](https://ardupilot.org/copter/docs/common-modalai-voxl.html) to make it clear that the VOXL2 works as well or we could make a new page.

**Comment by katzfey** [comment]
Oh, Lauren asked me to do this but I guess she already tried. Sorry for the confusion. But since VOXL 1 and VOXL 2 are very different boards it would be good to have a separate info for each one. And the current page features VOXL CAM (which, of course, has a VOXL 1 inside) so that's yet a different thing. I guess we should think through all this and come back with a better overall update. In the mean time though I think it would be good to just add this entry into the listing.

**Comment by rmackay9** [comment]
@katzfey,

If you guys are happy to send me a VOXL2, I can write the wiki page for you and it might also help with testing future updates (to AP or modalai).  No pressure though of course.  I can also be found in your Partners chat.  Thanks again!

**Comment by katzfey** [comment]
I added a little to that common page to reference VOXL 2. But, yes, it would be good to get you at least a VOXL 2 board.

File: common/source/docs/common-modalai-voxl.rst [hunk 1]
```diff
@@ -11,7 +11,8 @@ ModalAI VOXL
 
 This article explains how to setup a `ModalAI VOXL-CAM <https://www.modalai.com/en-jp/pages/voxl-cam-perception-engine>`__ for use with ArduPilot as a substitude for a GPS allowing position control modes like Loiter, PosHold, RTL, Auto to work.
 
-Other versions of the VOXL cameras should also work except for the VOXL2 which is not yet supported.
+The VOXL board in the VOXL-CAM can be purchased individually and used as a companion computer. The newer and more powerful VOXL 2 board also
```
[the remark this answers is not in the document]
Comment (katzfey) [reply common/source/docs/common-modalai-voxl.rst:14]: https://www.modalai.com/products/voxl-cam

[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-modalai-voxl.rst:14]: that is the camera +board, correct? the above implies just the companion computer board can be obtained....already this entire page is about the board+camera system...what I am missing?

[the remark this answers is not in the document]
Comment (katzfey) [reply common/source/docs/common-modalai-voxl.rst:14]: Oh, for just the VOXL bare board to be used as a companion computer this is the link: https://www.modalai.com/collections/voxl/products/voxl

[the remark this answers is not in the document]
Comment (katzfey) [reply common/source/docs/common-modalai-voxl.rst:14]: The voxl-cam just bundles that up with cameras into a case.

[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-modalai-voxl.rst:14]: thats what I needed to modify your PR.....thanks

[the remark this answers is not in the document]
Comment (katzfey) [reply common/source/docs/common-modalai-voxl.rst:14]: Is this ready to go now?

[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-modalai-voxl.rst:14]: not yet..will push some changes tomorrow...

[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-modalai-voxl.rst:14]: so you havent allowed others to modify...therefore I have to take another approach... I will merge this as is...and then immediately merge my modifications over it on the wiki...thanks for the update

[the remark this answers is not in the document]
Comment (katzfey) [reply common/source/docs/common-modalai-voxl.rst:14]: Oops, sorry, probably because it is from our fork. I'll submit future PRs as regular branches.

**Pull request merged**